Addition of NAK 3D Designs Edition

NAK3DDesigns opened the pull request.

NAK3DDesigns committed:
Create BB-NAK3D v61.step

Added NAK 3D Designs Edition

NAK3DDesigns committed:
Updated folder structure

Added NAK 3D folder and readme.

NAK3DDesigns committed:
Update README.md

NAK3DDesigns committed:
Updated Step file to V67

Minor tweeks, error corrections, and added cosmetic covers to steppers

NAK3DDesigns committed:
loaded lasted revision

many tweaks to include a bondtech style extruder.

NAK3DDesigns committed:
Update README.md

NAK3DDesigns committed:
Update README.md

pigeonladayyy commented:
I made a fork of the babybelt and just added the STL's as a ZIP in the same format as the Voron 2 package. I also made a BOM. I'm simple and cannot figure out how to do a merge of my version to yours. my work is attached,

[BB-NAK3D v81 STL's.zip](https://github.com/RobMink/Babybelt/files/8252575/BB-NAK3D.v81.STL.s.zip)
[BB-NAK3D v81_BOM.xlsx](https://github.com/RobMink/Babybelt/files/8252582/BB-NAK3D.v81_BOM.xlsx)

NAK3DDesigns commented:
> I made a fork of the babybelt and just added the STL's as a ZIP in the same format as the Voron 2 package. I also made a BOM. I'm simple and cannot figure out how to do a merge of my version to yours. my work is attached,
> 
> [BB-NAK3D v81 STL's.zip](https://github.com/RobMink/Babybelt/files/8252575/BB-NAK3D.v81.STL.s.zip) [BB-NAK3D v81_BOM.xlsx](https://github.com/RobMink/Babybelt/files/8252582/BB-NAK3D.v81_BOM.xlsx)

Please hold off on publishing STL's and BOM as the design is still in a constant state of change.  This is why we've only been publishing the step files..  For example you based this off v81 and I'm currently working on v101.  Once I have a finalized working and tested version I will publish all the STL files as well as a detailed BOM.

NAK3DDesigns committed:
Merge branch 'RobMink:main' into main

pigeonladayyy commented:
Ahh yep that makes sense. Cheers.

On Tue, 15 Mar 2022 at 22:31, NAK3DDesigns ***@***.***> wrote:

> I made a fork of the babybelt and just added the STL's as a ZIP in the
> same format as the Voron 2 package. I also made a BOM. I'm simple and
> cannot figure out how to do a merge of my version to yours. my work is
> attached,
>
> BB-NAK3D v81 STL's.zip
> <https://github.com/RobMink/Babybelt/files/8252575/BB-NAK3D.v81.STL.s.zip> BB-NAK3D
> v81_BOM.xlsx
> <https://github.com/RobMink/Babybelt/files/8252582/BB-NAK3D.v81_BOM.xlsx>
>
> Please old off on publishing STL's and BOM as the design is still in a
> constant state of change. This is why we've only been publishing the step
> files.. For example you based this off v81 and I'm currently working on
> v101.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RobMink/Babybelt/pull/1#issuecomment-1067883853>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYHN5EJH7RE7DFFP7LIY4DTVABYJ7ANCNFSM5PI2LIGQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

pigeonladayyy commented:
Do you want me to retract it or should I leave it in? Sorry for the hassle.

On Wed, 16 Mar 2022 at 11:57, Sam Flynn ***@***.***> wrote:

> Ahh yep that makes sense. Cheers.
>
> On Tue, 15 Mar 2022 at 22:31, NAK3DDesigns ***@***.***>
> wrote:
>
>> I made a fork of the babybelt and just added the STL's as a ZIP in the
>> same format as the Voron 2 package. I also made a BOM. I'm simple and
>> cannot figure out how to do a merge of my version to yours. my work is
>> attached,
>>
>> BB-NAK3D v81 STL's.zip
>> <https://github.com/RobMink/Babybelt/files/8252575/BB-NAK3D.v81.STL.s.zip> BB-NAK3D
>> v81_BOM.xlsx
>> <https://github.com/RobMink/Babybelt/files/8252582/BB-NAK3D.v81_BOM.xlsx>
>>
>> Please old off on publishing STL's and BOM as the design is still in a
>> constant state of change. This is why we've only been publishing the step
>> files.. For example you based this off v81 and I'm currently working on
>> v101.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/RobMink/Babybelt/pull/1#issuecomment-1067883853>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AYHN5EJH7RE7DFFP7LIY4DTVABYJ7ANCNFSM5PI2LIGQ>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you commented.Message ID:
>> ***@***.***>
>>
>

NAK3DDesigns committed:
Updated to V107

updated step to V107 and updated readme with new features.

NAK3DDesigns committed:
Updated to V110

Updated to version V110.  V110 updates the cooling fan shroud and the worm drive gear (aka the one attached to the roller)

NAK3DDesigns committed:
Update README.md

NAK3DDesigns committed:
Update README.md
